### PR TITLE
README: fix spelling errors

### PR DESCRIPTION
**Pull request by yegorich**
Signed-off-by: Yegor Yefremov <yegorslists@googlemail.com>

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -107,14 +107,14 @@ lib/include directory. There are also test programs in the test/
 directory which give examples of use.
 
 Python bindings are in bindings/python and are commented in a similar
-manner. There is also test code in the test/ directory which give
+manner. There is also a test code in the test/ directory which gives
```
Comment (jackmitch) [inline README]: this was right the first time round.

File: README [hunk 2]
```diff
@@ -107,14 +107,14 @@ lib/include directory. There are also test programs in the test/
...
 examples of use.
 
 Can I help?
 ------------------------------------------------------------------------
 
-libsoc is Open Source Software and as such you are welcome to browse the
-code and either add features, or fix bugs. Please submit pull requests,
+libsoc is an Open Source Software and as such you are welcome to browse
```
Comment (jackmitch) [inline README]: this 'an' isn't right here

File: README (hunk 1 above)
Comment (yegorich) [inline README]: OK then I'll drop 'a'.

File: README (hunk 2 above)
Comment (yegorich) [inline README]: And also drop this change.

Comment (jackmitch) [inline README]: 'libsoc is open-source software' would probably be more gramatically correct if you want to switch to it

File: README (hunk 1 above)
Comment (jackmitch) [inline README]: I'm 90% sure the 'give' is right too, but I might need to consult someone with better grammar skills than myself to be sure

Comment (jackmitch) [inline README]: https://books.google.co.uk/books?id=kknpPZjUR5IC&pg=PA321&lpg=PA321&dq=%22give+examples+of+use%22&source=bl&ots=EmY0h3IaIW&sig=MQGBfjFNhMSI0RBgdipdYTEyWUM&hl=en&sa=X&ved=0ahUKEwi7kbTZzrvMAhWB2BoKHbBjD6sQ6AEILDAD#v=onepage&q=%22give%20examples%20of%20use%22&f=false

seems like a good enough example of its correctness

Comment (yegorich) [inline README]: No, because we give examples, i.e. plural and in our case the code gives examples, i.e. third person singular or do I see it wrong?

Comment (jackmitch) [inline README]: Just done some more reading and I think I agree, it should be gives as 'code' is third person like you say, good spot!

**Pull request closed without merging**